nemo/utils/dllogger.py

```diff
@@ -31,6 +30,13 @@
 except (ImportError, ModuleNotFoundError):
     HAVE_DLLOGGER = False
 
+try:
+    from lightning_fabric.utilities.logger import _convert_params, _flatten_dict, _sanitize_callable_params
+
+    PL_LOGGER_UTILITIES = True
+except (ImportError, ModuleNotFoundError):
+    PL_LOGGER_UTILITIES = False
```

Comment:
Not the cleanest solution but prevents us touching internals within Lightning (and support 1.8.x and 1.9.x).

I told the Lightning team that they should consider putting these protected variables in their Logger class, so that other loggers can use these functions without calling internals.